classes/image_types_ostree.bbclass

```diff
@@ -41,7 +41,7 @@ IMAGE_CMD_ostreecredunpack () {
 	if [ ${SOTA_PACKED_CREDENTIALS} ]; then
 		rm -rf ${DEPLOY_DIR_IMAGE}/sota_credentials
 
-		unzip ${SOTA_PACKED_CREDENTIALS} -d ${DEPLOY_DIR_IMAGE}/sota_credentials
+		${bindir}/unzip ${SOTA_PACKED_CREDENTIALS} -d ${DEPLOY_DIR_IMAGE}/sota_credentials
```

Comment:
I have updated the patch and added unzip-native as a dependency. 

Btw according [to the mega manual of the Yocto Project for Pyro](http://www.yoctoproject.org/docs/2.3/ref-manual/ref-manual.html#required-packages-for-the-host-development-system) (aka version 2.3) unzip is part of the essential packages that have to be installed on the host.